packages/app-store/make/lib/nodeScheduler.ts

```diff
@@ -0,0 +1,94 @@
+import schedule from "node-schedule";
+
+import prisma from "@calcom/prisma";
+
+export async function scheduleTrigger(
```

Comment:
This looks like the exact same we have for zapier. We should definitely reuse that as well